02-inspect-file.md

```diff
@@ -169,7 +169,7 @@ Instead of simply counting the files that match the search criteria, save them t
 This is done with the redirection operator, `>`.
 
 ~~~ {.bash}
-gunzip -c counts-raw.txt.gz | grep "Evolutionary Biology" | grep "Cell Biology" > evo-cell-bio.txt
+zcat counts-raw.txt.gz | cut -f11 | grep "Evolutionary Biology" | grep "Cell Biology" > evo-cell-bio.txt
```

Comment:
I went in and reverted it, but I had already gone in and refixed it. So,
after some git wrangling, I think it ended up in the right place, but if it
isn't automatically fixed in the PR I can go back in and do something
different. Sorry!

On Mon, Sep 14, 2015 at 11:23 PM, John Blischak <notifications@github.com>
wrote:

> In 02-inspect-file.md
> <https://github.com/jdblischak/r-intermediate-altmetrics/pull/3#discussion_r39471208>
> :
>
> > @@ -169,7 +169,7 @@ Instead of simply counting the files that match the search criteria, save them t
> >  This is done with the redirection operator, `>`.
> >
> >  ~~~ {.bash}
> > -gunzip -c counts-raw.txt.gz | grep "Evolutionary Biology" | grep "Cell Biology" > evo-cell-bio.txt
> > +zcat counts-raw.txt.gz | cut -f11 | grep "Evolutionary Biology" | grep "Cell Biology" > evo-cell-bio.txt
>
> I think it would be easiest if you just reverted that one commit and then
> pushed it:
>
> git revert e103fb82612e1629a9c0db11a0ece205cf4c8c99
> git push origin gh-pages
>
> The PR will be automatically updated with this new commit that reverses
> that previous one.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jdblischak/r-intermediate-altmetrics/pull/3/files#r39471208>
> .
>